src/components/transaction-list/index.jsx

```diff
@@ -20,17 +21,21 @@ dayjs.extend(calendar);
 export const TransactionList = ({ primary, count, className }) => {
```

Comment:
Do we still need `count`?

Reply:
Why do you pass it inside the method if the actual number is always 10?

Reply:
Display block "The last 10 transactions are shown here" and limits in the request